python/hsfs/training_dataset.py

```diff
@@ -125,8 +124,8 @@ def __init__(
             self._training_dataset_type = None
         # set up depending on user initialized or coming from backend response
         if created is None:
-            self._start_time = self._convert_event_time_to_timestamp(event_start_time)
-            self._end_time = self._convert_event_time_to_timestamp(event_end_time)
+            self._start_time = util.convert_event_time_to_timestamp(event_start_time)
```

Comment:
Maybe we should do the conversion as well in `_set_time_splits`

Reply:
done